fabrictestbed_extensions/fablib/network_service.py

```diff
@@ -735,6 +743,31 @@ def get_gateway(self) -> IPv4Address or IPv6Address or None:
             logging.warning(f"Failed to get gateway: {e}")
             return None
 
+    # def change_public_ip(self, ipv6: list[str] = None, ipv4: list[str] = None):
```

Comment:
Is this intentionally commented?

Reply:
This is commented out for right now as per our conversation on slack. It is intentional but will be added later on once I add everything required.